package.json

```diff
@@ -15,6 +15,9 @@
     "redux-saga": "0.16.0",
     "redux-thunk": "2.2.0"
   },
+  "engines": {
+    "node": ">=8"
+  },
```

Comment:
Why is that needed? Also, why node 9 (which is the latest version i believe?)

Reply:
The `config-overrides.js` file doesn't work with `node<8` this is to make sure yarn warns us about it if we are still using node6